bedrock/settings/base.py

```diff
@@ -1529,3 +1529,6 @@ def get_default_gateway_linux():
 
 # Issue 7508 - Convert.com experiment sandbox
 CONVERT_PROJECT_ID = ('10039-1003350' if DEV else '10039-1003343')
+
+# Issue 8444 - Sentry Client side SDK
+SENTRY_DATA_SOURCE_NAME = config('SENTRY_DATA_SOURCE_NAME', default='')
```

Comment:
@pmac I took a pop at adding this to https://www-demo1.allizom.org/en-US/

Do we already have the Sentry DSN's for each deployment set up as environment variables?

Reply:
We do, but I think the one for the client-side is different. The server side one is secret.

https://github.com/mozilla/bedrock/blob/1cd982199cd19dfdca7fd8fd5cdd78309f94b4d7/bedrock/settings/base.py#L1434